modules/bluefish/util/blue_velvet.cpp

```diff
@@ -20,67 +20,231 @@
+			GET_PROCADDR_FOR_FUNC(bfcGetCardSerialNumber, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetCardFwVersion, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcWaitVideoSyncAsync, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcWaitVideoInputSync, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcWaitVideoOutputSync, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetVideoOutputCurrentFieldCount, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetVideoInputCurrentFieldCount, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcVideoCaptureStart, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcVideoCaptureStop, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcVideoPlaybackStart, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcVideoPlaybackStop, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcVideoPlaybackAllocate, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcVideoPlaybackPresent, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcVideoPlaybackRelease, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetCaptureVideoFrameInfoEx, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcRenderBufferCapture, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcRenderBufferUpdate, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetRenderBufferCount, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcEncodeHancFrameEx, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcDecodeHancFrameEx, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcSystemBufferReadAsync, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcSystemBufferWriteAsync, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetBytesForGroupPixels, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetPixelsPerLine, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetLinesPerFrame, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetBytesPerLine, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetBytesPerFrame, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetGoldenValue, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetVBILines, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetVANCGoldenValue, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetVANCLineBytes, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetVANCLineCount, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcGetWindowsDriverHandle, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetStringForCardType, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetStringForBlueProductId, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetStringForVideoMode, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetStringForMemoryFormat, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetMR2Routing, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsSetMR2Routing, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetAudioOutputRouting, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsSetAudioOutputRouting, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsIsVideoModeProgressive, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsIsVideoMode1001Framerate, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetFpsForVideoMode, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetVideoModeForFrameInfo, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetFrameInfoForVideoMode, hModule);
+			GET_PROCADDR_FOR_FUNC(bfcUtilsGetAudioSamplesPerFrame, hModule);
+
+			bRes = true;
+		}
+
+		return bRes;
+	}
+
+	bool BvcWrapper::IsBvcValid()
```

Comment:
Should be removed completely after the changes in my other comments, `bvc` should never be `nullptr` if the `BvcWrapper` has been contstructed.